modules/nw-egress-ips-about.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/assigning-egress-ips.adoc
+
+[id="nw-egress-ips-about_{context}"]
+= Egress IP address assignment for project egress traffic
+
+By configuring an egressIP for a project, all outgoing external connections from the specified project will share the same, fixed source IP address.
```

Comment:
s/egressIP/egress IP (Alternatively, if this is a variable, you can set it in backticks)

Reply:
Oops, supposed to just be _egress IP address_.